### PR TITLE
Add support for Mustache templates in comments

### PR DESCRIPTION
**Pull request by angelikatyborska**
Closes https://github.com/exercism/pr-commenter-action/issues/8

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -109,6 +109,49 @@ The text to be included in the PR comment.
...
+          echo ::set-output name=today::"$(date +%d-%m-%Y)"
+
+      - uses: exercism/pr-commenter-action@v1.1.0
+        with:
+          template-variables: |
+            {
+              "today": "${{ steps.vars.outputs.today }}",
+              "prAuthor": "${{ github.event.pull_request.user.login }}",
+              "branchNamePrefix": ${{ startsWith(github.event.pull_request.head.ref, 'ref-') }}
+            }
```
Comment (angelikatyborska) [inline README.md:143]: Is there a nicer way to pass an object as input?

**Comment by angelikatyborska** [comment]
@SleeplessByte @SaschaMann It would be awesome if you could review this.

I already tested all the expressions I added to the README in a test repo , so I'm pretty sure they're correct, but I would appreciate a sanity check and general opinions.

File: README.md (hunk 1 above)
Comment (SaschaMann) [reply README.md:143]: I don't know of one.

You could pass them as environment variables with a predetermined prefix, e.g.

```yaml
env:
  VAR_PR_AUTHOR: ${{ github.event.pull_request.user.login }}
  VAR_TODAY: ${{ steps.vars.outputs.today }}
  VAR_BRANCH_NAME_PREFIX: ${{ startsWith(github.event.pull_request.head.ref, 'ref-') }}
```

or, if you don't want a prefix, allow all environment variables as inputs for the template.

**Comment by angelikatyborska** [comment]
It's been 2 weeks and I see no objections, so I'll just merge and release.

**Pull request merged**